pom.xml

```diff
@@ -144,11 +144,6 @@
           <extensions>true</extensions>
           <configuration>
             <instructions>
-              <_exportcontents>
-                io.cdap.plugin.db.batch.action.vertica.export.*;
```

Comment:
this line still needs to be exported (it's where the plugin lives). The 2 lines below are the ones that are not needed

Reply:
Done.